recipes/gcc/all/conanfile.py

```diff
@@ -108,9 +110,10 @@ def package(self):
             autotools.install(args=self._make_args)
         else:
             autotools.make(args=["install-strip"] + self._make_args)
-        tools.rmdir(os.path.join(self.package_folder, "share"))
-        tools.remove_files_by_mask(self.package_folder, "*.la")
-        self.copy(pattern="COPYING*", dst="licenses", src=self._source_subfolder)
+        rmdir(self, os.path.join(self.package_folder, "share"))
+        rm(self, "*.la", self.package_folder)
```

Comment:
```suggestion
        rm(self, "*.la", self.package_folder, recursive=True)
```
Need to go deeper to find .la files